// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Regression testing  should match for data copied from a test run with quarantine mode on 1`] = `
<?xml version="1.0" encoding="UTF-8" ?>
<test-run id="2" name="TestCafe Tests" start-time="1970-03-14T15:27:12.448Z" end-time="1970-03-14T15:28:03.175Z">
    <test-suite type="Assembly" name="Banana" start-time="2077-03-14T03:14:15.926Z" end-time="2077-03-14T03:14:15.926Z">
        <test-case name="Split" fullname="Banana - Split" result="Failed" duration="17.2">
            <failure>
                <message>
                    <![CDATA[AssertionError: expected '41' to deeply equal '7'

Run 1: Failed
Run 2: Failed
Run 3: Failed]]>
                </message>
                <stack-trace>
                    <![CDATA[-- Error on page "http://example.org":
⠀
   AssertionError: expected '41' to deeply equal '7'
⠀
   Browser: Chrome 80.0.3987.132 / Windows 10
   Screenshot: C:\\testcafe-demo\\screenshots\\1970-03-14_11-27-11\\test-1\\errors\\1_run-3_Chrome.png]]>
                </stack-trace>
            </failure>
            <attachments>
                <attachment>
                    <filePath>C:\\testcafe-demo\\screenshots\\1970-03-14_11-27-11\\test-1\\errors\\1_run-1_Chrome.png</filePath>
                </attachment>
                <attachment>
                    <filePath>C:\\testcafe-demo\\screenshots\\1970-03-14_11-27-11\\test-1\\errors\\1_run-2_Chrome.png</filePath>
                </attachment>
                <attachment>
                    <filePath>C:\\testcafe-demo\\screenshots\\1970-03-14_11-27-11\\test-1\\errors\\1_run-3_Chrome.png</filePath>
                </attachment>
            </attachments>
        </test-case>
        <test-case name="Costume" fullname="Banana - Costume" result="Failed" duration="15.978">
            <failure>
                <message>
                    <![CDATA[AssertionError: expected false to be truthy

Run 1: Failed
Run 2: Failed
Run 3: Failed]]>
                </message>
                <stack-trace>
                    <![CDATA[-- Error on page "http://example.org":
⠀
   AssertionError: expected false to be truthy
⠀
   Browser: Chrome 80.0.3987.132 / Windows 10
⠀
   ⠀  1 |import * as createCallsiteRecord from 'callsite-record';
   ⠀  2 |
   ⠀  3 |function functionThatThrowsUp() {
   ⠀> 4 |  throw new Error('Hey ya!');
   ⠀  5 |}
   ⠀  6 |
   ⠀  7 |try {
   ⠀  8 |  functionThatThrowsUp();
   ⠀  9 |} catch (err) {
⠀
      at functionThatThrowsUp (some-file:1:1)
      at Object.<anonymous> (some-file:1:1)
      at Runtime._execModule (some-file:1:1)
      at Runtime._loadModule (some-file:1:1)
      at Runtime.requireModule (some-file:1:1)
      at Runtime.requireModuleOrMock (some-file:1:1)
      at Object.<anonymous> (some-file:1:1)
      at Runtime._execModule (some-file:1:1)
      at Runtime._loadModule (some-file:1:1)
      at Runtime.requireModule (some-file:1:1)]]>
                </stack-trace>
            </failure>
            <attachments>
                <attachment>
                    <filePath>C:\\testcafe-demo\\screenshots\\1970-03-14_11-27-11\\test-2\\errors\\1_run-1_Chrome.png</filePath>
                </attachment>
                <attachment>
                    <filePath>C:\\testcafe-demo\\screenshots\\1970-03-14_11-27-11\\test-2\\errors\\1_run-2_Chrome.png</filePath>
                </attachment>
                <attachment>
                    <filePath>C:\\testcafe-demo\\screenshots\\1970-03-14_11-27-11\\test-2\\errors\\1_run-3_Chrome.png</filePath>
                </attachment>
            </attachments>
        </test-case>
    </test-suite>
    <test-suite type="Assembly" name="Pear" start-time="2077-03-14T03:14:15.926Z" end-time="2077-03-14T03:14:15.926Z">
        <test-case name="Fast" fullname="Pear - Fast" result="Passed" duration="0.716">
        </test-case>
        <test-case name="Faster" fullname="Pear - Faster" result="Failed" duration="16.582">
            <failure>
                <message>
                    <![CDATA[AssertionError: expected '41' to deeply equal '5'

Run 1: Failed
Run 2: Failed
Run 3: Failed]]>
                </message>
                <stack-trace>
                    <![CDATA[-- Error on page "http://example.org":
⠀
   AssertionError: expected '41' to deeply equal '5'
⠀
   Browser: Chrome 80.0.3987.132 / Windows 10
   Screenshot: C:\\testcafe-demo\\screenshots\\1970-03-14_11-27-11\\test-4\\errors\\1_run-3_Chrome.png]]>
                </stack-trace>
            </failure>
            <attachments>
                <attachment>
                    <filePath>C:\\testcafe-demo\\screenshots\\1970-03-14_11-27-11\\test-4\\errors\\1_run-1_Chrome.png</filePath>
                </attachment>
                <attachment>
                    <filePath>C:\\testcafe-demo\\screenshots\\1970-03-14_11-27-11\\test-4\\errors\\1_run-2_Chrome.png</filePath>
                </attachment>
                <attachment>
                    <filePath>C:\\testcafe-demo\\screenshots\\1970-03-14_11-27-11\\test-4\\errors\\1_run-3_Chrome.png</filePath>
                </attachment>
            </attachments>
        </test-case>
    </test-suite>
</test-run>
`;

exports[`Regression testing  should match for the example data used by the official TestCafe reporters 1`] = `
<?xml version="1.0" encoding="UTF-8" ?>
<test-run id="2" name="TestCafe Tests" start-time="1970-01-01T00:00:00.000Z" end-time="1970-01-01T00:15:25.000Z">
    <test-suite type="Assembly" name="First fixture" start-time="2077-03-14T03:14:15.926Z" end-time="2077-03-14T03:14:15.926Z">
        <test-case name="First test in first fixture" fullname="First fixture - First test in first fixture" result="Inconclusive" duration="74">
            <attachments>
                <attachment>
                    <filePath>C:/screenshots/1445437598847/1.png</filePath>
                </attachment>
                <attachment>
                    <filePath>C:/screenshots/1445437598847/2.png</filePath>
                </attachment>
            </attachments>
        </test-case>
        <test-case name="Second test in first fixture" fullname="First fixture - Second test in first fixture" result="Failed" duration="74">
            <failure>
                <message>
                    <![CDATA[Some error
]]>
                </message>
                <stack-trace>
                    <![CDATA[-- Error on page "http://example.org":
⠀
   Some error
⠀
   Browser: Chrome 41.0.2227 / Mac OS X 10.10.1
⠀
   ⠀  1 |import * as createCallsiteRecord from 'callsite-record';
   ⠀  2 |
   ⠀  3 |function functionThatThrowsUp() {
   ⠀> 4 |  throw new Error('Hey ya!');
   ⠀  5 |}
   ⠀  6 |
   ⠀  7 |try {
   ⠀  8 |  functionThatThrowsUp();
   ⠀  9 |} catch (err) {
⠀
      at functionThatThrowsUp (some-file:1:1)
      at Object.<anonymous> (some-file:1:1)
      at Runtime._execModule (some-file:1:1)
      at Runtime._loadModule (some-file:1:1)
      at Runtime.requireModule (some-file:1:1)
      at Runtime.requireModuleOrMock (some-file:1:1)
      at Object.<anonymous> (some-file:1:1)
      at Runtime._execModule (some-file:1:1)
      at Runtime._loadModule (some-file:1:1)
      at Runtime.requireModule (some-file:1:1)
⠀
-- The specified selector does not match any element in the DOM tree.
⠀
   Browser: Firefox 47 / Mac OS X 10.10.1
⠀
   ⠀  1 |import * as createCallsiteRecord from 'callsite-record';
   ⠀  2 |
   ⠀  3 |function functionThatThrowsUp() {
   ⠀> 4 |  throw new Error('Hey ya!');
   ⠀  5 |}
   ⠀  6 |
   ⠀  7 |try {
   ⠀  8 |  functionThatThrowsUp();
   ⠀  9 |} catch (err) {
⠀
      at functionThatThrowsUp (some-file:1:1)
      at Object.<anonymous> (some-file:1:1)
      at Runtime._execModule (some-file:1:1)
      at Runtime._loadModule (some-file:1:1)
      at Runtime.requireModule (some-file:1:1)
      at Runtime.requireModuleOrMock (some-file:1:1)
      at Object.<anonymous> (some-file:1:1)
      at Runtime._execModule (some-file:1:1)
      at Runtime._loadModule (some-file:1:1)
      at Runtime.requireModule (some-file:1:1)]]>
                </stack-trace>
            </failure>
        </test-case>
        <test-case name="Third test in first fixture" fullname="First fixture - Third test in first fixture" result="Passed" duration="74">
        </test-case>
    </test-suite>
    <test-suite type="Assembly" name="Second fixture" start-time="2077-03-14T03:14:15.926Z" end-time="2077-03-14T03:14:15.926Z">
        <test-case name="First test in second fixture" fullname="Second fixture - First test in second fixture" result="Passed" duration="74">
        </test-case>
        <test-case name="Second test in second fixture" fullname="Second fixture - Second test in second fixture" result="Passed" duration="74">
        </test-case>
        <test-case name="Third test in second fixture" fullname="Second fixture - Third test in second fixture" result="Passed" duration="0">
        </test-case>
    </test-suite>
    <test-suite type="Assembly" name="Third fixture" start-time="2077-03-14T03:14:15.926Z" end-time="2077-03-14T03:14:15.926Z">
        <test-case name="First test in third fixture" fullname="Third fixture - First test in third fixture" result="Failed" duration="74">
            <failure>
                <message>
                    <![CDATA[]]>
                </message>
                <stack-trace>
                    <![CDATA[-- - Error in beforeEach hook -
   The specified selector does not match any element in the DOM tree.
⠀
   Browser: Firefox 47 / Mac OS X 10.10.1
⠀
   ⠀  1 |import * as createCallsiteRecord from 'callsite-record';
   ⠀  2 |
   ⠀  3 |function functionThatThrowsUp() {
   ⠀> 4 |  throw new Error('Hey ya!');
   ⠀  5 |}
   ⠀  6 |
   ⠀  7 |try {
   ⠀  8 |  functionThatThrowsUp();
   ⠀  9 |} catch (err) {
⠀
      at functionThatThrowsUp (some-file:1:1)
      at Object.<anonymous> (some-file:1:1)
      at Runtime._execModule (some-file:1:1)
      at Runtime._loadModule (some-file:1:1)
      at Runtime.requireModule (some-file:1:1)
      at Runtime.requireModuleOrMock (some-file:1:1)
      at Object.<anonymous> (some-file:1:1)
      at Runtime._execModule (some-file:1:1)
      at Runtime._loadModule (some-file:1:1)
      at Runtime.requireModule (some-file:1:1)]]>
                </stack-trace>
            </failure>
        </test-case>
    </test-suite>
</test-run>
`;
